honeycomb-core/src/cmap/components/identifiers.rs

```diff
@@ -0,0 +1,96 @@
+// ------ IMPORTS
+
+// ------ CONTENT
+
+macro_rules! impl_from_dartid {
+    ($idty: ty) => {
+        impl From<DartId> for $idty {
+            fn from(dart_id: DartId) -> Self {
+                Self(dart_id.0.into())
+            }
+        }
+    };
+}
+
+macro_rules! impl_from_for_dartid {
+    ($idty: ty) => {
+        impl From<$idty> for DartId {
+            fn from(id: $idty) -> Self {
+                Self(id.0.into())
+            }
+        }
+    };
+}
```

Comment:
Is is useful?

Reply:
1. the main use case is to have a clean API when building orbits for all cells of a kind, e.g. fetch all vertex ID (a vector), and compute all darts of each vertex (turns into a matrix). This is done using `Orbit2` which takes a dart id as argument atm. I can change its API to take anything that implements `Into<DartIdentifier>`
2. there should be no problem with regular integers since these `From` impl only convert IDs between each other (explictly), they are not implemented for integer primitives